app/utils.py

```diff
@@ -7,19 +7,20 @@
 from .config import WORDS_FILE_PATH
 from .log import LOG
 
-with open(WORDS_FILE_PATH) as f:
-    LOG.d("load words file: %s", WORDS_FILE_PATH)
-    _words = f.read().split()
+def _word():
```

Comment:
Actually the reason `_words` was loaded at module level before is to avoid calling `random_word()` or `random_words` loads the `WORDS_FILE_PATH` again. If `WORDS_FILE_PATH` is a big file, then the loading time could be expensive.